<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleNetworkWindow</class>
 <widget class="QWidget" name="BattleNetworkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BattleShip Network Player Lobby</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>389</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="NetworkPlayersList"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="PlayWithButton">
        <property name="text">
         <string>Start Game</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ReturnToButton">
        <property name="text">
         <string>Return To Menu</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="GameLogo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BattleShip</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
